website/docs/patterns/table-multi-select/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,33 @@
+!!! Info
+
+It is the responsibility of consumers to implement this pattern, and therefore also their responsibility to ensure that all accessibility success criteria are met or exceeded.
+!!!
+
+## Aria role
```

Comment:
@MelSumner Could you have a look at this to make sure the language I'm using is correct?

Reply:
Can this be resolved now?

Reply:
Yes, I think so.